exasol/ds/sandbox/lib/asset_printing/mark_down_printer.py

```diff
@@ -28,18 +36,38 @@ def finish(self):
 
 
 class MarkdownTextPrinter(TextPrinter, ABC):
+    def __init__(self, target: io.TextIOBase):
+        self.target = target
+
+    def _format(self, obj: TextObject) -> Optional[str]:
+        if isinstance(obj, TitleTextObject):
+            return f'#### {obj.text}\n'
+        if isinstance(obj, CodeBlockTextObject):
+            return cleandoc(f"""
+            ```shell
```

Comment:
If we have different source languages, we might need to add this to `CodeBlockTextObject`

Reply:
Will do this on demand